README.txt

```diff
@@ -97,6 +97,12 @@ has variant name ``rs75454623``::
 
     $ ga4gh_client variants-search http://localhost:8000 -V 1000g_2013 -r 1 -e 60000 -n rs75454623  | less -S
 
+we can also run the "callset/search" method over all the variant sets in ga4gh-example-data,
```

Comment:
I think we can remove this paragraph --- it's not essential to get people started.